### PR TITLE
CameraRoll mock

### PR DESCRIPTION
**Pull request by ivanbtrujillo**
This PR adds mock data to cameraRoll in order to allow testing of components that uses cameraRoll.getPhotos method.

- Add mocked data to getPhotos method of CameraRollManager (NativeModule)
- Add a test to check that getPhotos method of CameraRoll receive that data.

An example:

A component which have a getImages method like this:
```javascript
  getPhotos = async (params) => {
    const photos = await new Promise((res, rej) =>
      CameraRoll.getPhotos(params)
        .then(({ edges, page_info }) => {
          const images = edges.map(asset => asset.node.image)
          res({ images, page_info })
        })
        .catch(rej))
    return photos
  }
```

Could be tested like this:
```javascript
test('getPhotos(): should get images from gallery', async () => {
    const wrapper = shallow(<PhotoGallery />)
    const expectedResult = {
      images: [
        {
          uri: 'content://media/external/images/media/1',
          height: 2448,
          width: 3968,
        },
        {
          uri: 'content://media/external/images/media/2',
          height: 2448,
          width: 3968,
        },
        {
          uri: 'content://media/external/images/media/3',
          height: 2448,
          width: 3968,
        },
      ],
      page_info: {
        has_next_page: true,
        end_cursor: '1528919312601',
      },
    }
    const result = await wrapper.instance().getPhotos()

    wrapper.update()

    expect(result.images.length).toBe(3)
    expect(result.page_info.has_next_page).toBe(true)
    expect(result).toMatchObject(expectedResult)
  })
```

### CODE REVIEW

**Comment by ivanbtrujillo** [comment]
@johnstem updated!

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@ build/Release
 
 # Dependency directory
 node_modules
+package-lock.json
```
Comment (bob-carson) [inline .gitignore]: Hi @ivanbtrujillo, thanks for opening this PR.  Is there a need to add this to our `.gitignore` and if not, can you update your change to not include this?

Comment (ivanbtrujillo) [reply .gitignore]: The reason is that in the previous comment @johnstem ask for remove package-lock.json. 

I've read that is recommended to keep it  on the repo but I don't know the reasons that maintainers have to remove it.

Should I keep it or not? 

Thanks!

Comment (bob-carson) [reply .gitignore]: If we are removing the file we should remove this line from the `.gitignore` as well.  

If you get that change committed and make sure the linter and test suite are passing I can run this again our projects and get it merged.

**Comment by bob-carson** [comment]
I went ahead and made that change and merged this in https://github.com/Root-App/react-native-mock-render/pull/56

**Pull request merged**